src/scss/styles.css

```diff
@@ -5,81 +5,252 @@ All Styles
 body {
```

Comment:
I’m surprised to see a CSS file in the SCSS folder. Did you add this, or did the build system?